pkg/machinery/resources/network/link.go

```diff
@@ -50,6 +50,11 @@ type BondMasterSpec struct {
 	PeerNotifyDelay uint32                        `yaml:"peerNotifyDelay,omitempty"`
 }
 
+// BridgeMasterSpec describes bridge settings if Kind == "bridge".
+type BridgeMasterSpec struct {
+	STPEnabled bool `yaml:"stpEnabled,omitempty"`
```

Comment:
same as in  the machine config, we might better structure it as `STP STPSpec ...` so that we can add more STP settings

Reply:
Done.